source/scpp/src/crypto/ByteSliceHasher.cpp

```diff
@@ -9,7 +9,7 @@ namespace stellar
 {
 namespace shortHash
 {
-static unsigned char sKey[crypto_shorthash_KEYBYTES];
+thread_local unsigned char sKey[crypto_shorthash_KEYBYTES];
```

Comment:
Is the initialize function ever called ?

Reply:
It looks like it's not. It seems odd it was initialized to a random value anyway..